diff --git a/langs7/fr_FR.js b/langs7/fr_FR.js
index 24698d2..762e89e 100644
--- a/langs7/fr_FR.js
+++ b/langs7/fr_FR.js
@@ -284,7 +284,7 @@ tinymce.addI18n("fr_FR", {
     "Range 0 to 255": "Plage de 0 \xe0 255",
     "Red": "Rouge",
     "Red component": "Composant rouge",
-    "Redo": "R\xe9o\xe9ter",
+    "Redo": "Refaire",
     "Remove": "Retir\xe9",
     "Remove color": "Supprimer la couleur",
     "Remove link": "Enlever le lien",
diff --git a/langs7/zh_TW.js b/langs7/zh_TW.js
index fe9faad..eb221a2 100644
--- a/langs7/zh_TW.js
+++ b/langs7/zh_TW.js
@@ -284,7 +284,7 @@ tinymce.addI18n("zh_TW", {
     "Range 0 to 255": "\u7bc4\u570d0\u81f3255",
     "Red": "\u7d05\u8272",
     "Red component": "\u7d05\u8272\u90e8\u5206",
-    "Redo": "Test",
+    "Redo": "\u91cd\u505a",
     "Remove": "\u79fb\u9664",
     "Remove color": "\u79fb\u9664\u984f\u8272",
     "Remove link": "\u79fb\u9664\u9023\u7d50",
