From b1a575826e966b275eeed7585480244e3e0e8408 Mon Sep 17 00:00:00 2001
From: Erik Kjosavik <erik.kjosavik@gmail.com>
Date: Wed, 21 Jun 2023 15:41:09 +0200
Subject: [PATCH] Update langs

---
 langs6/nb_NO.js | 30 +++++++++++++++---------------
 1 file changed, 15 insertions(+), 15 deletions(-)

diff --git a/langs6/nb_NO.js b/langs6/nb_NO.js
index 6f5cd39..31d7e41 100644
--- a/langs6/nb_NO.js
+++ b/langs6/nb_NO.js
@@ -344,35 +344,35 @@ tinymce.addI18n("nb_NO", {
     "B": "B",
     "Blue component": "Bl\xe5 komponent",
     "#": "#",
-    "Hex color code": "Hex fargekode",
+    "Hex color code": "Heksadesimal fargekode",
     "Range 0 to 255": "Omr\xe5de 0 til 255",
     "Turquoise": "Turkis",
     "Green": "Gr\xf8nn",
     "Blue": "Bl\xe5",
     "Purple": "Lilla",
     "Navy Blue": "Marinebl\xe5",
-    "Dark Turquoise": "M\xf8rk turkis",
+    "Dark Turquoise": "M\xf8rketurkis",
     "Dark Green": "M\xf8rkegr\xf8nn",
     "Medium Blue": "Mellombl\xe5",
-    "Medium Purple": "Medium lilla",
+    "Medium Purple": "Mellomlilla",
     "Midnight Blue": "Midnattbl\xe5",
     "Yellow": "Gul",
     "Orange": "Oransje",
     "Red": "R\xf8d",
-    "Light Gray": "Lys gr\xe5",
+    "Light Gray": "Lysegr\xe5",
     "Gray": "Gr\xe5",
-    "Dark Yellow": "M\xf8rk gul",
-    "Dark Orange": "M\xf8rk oransje",
+    "Dark Yellow": "M\xf8rkegul",
+    "Dark Orange": "M\xf8rkeoransje",
     "Dark Red": "M\xf8rker\xf8d",
-    "Medium Gray": "Medium gr\xe5",
-    "Dark Gray": "M\xf8rk gr\xe5",
-    "Light Green": "Lys gr\xf8nn",
-    "Light Yellow": "Lys gul",
-    "Light Red": "Lys r\xf8d",
-    "Light Purple": "Lys lilla",
-    "Light Blue": "Lys bl\xe5",
-    "Dark Purple": "M\xf8rk lilla",
-    "Dark Blue": "M\xf8rk bl\xe5",
+    "Medium Gray": "Mellomgr\xe5",
+    "Dark Gray": "M\xf8rkegr\xe5",
+    "Light Green": "Lysegr\xf8nn",
+    "Light Yellow": "Lysegul",
+    "Light Red": "Lyser\xf8d",
+    "Light Purple": "Lyselilla",
+    "Light Blue": "Lysebl\xe5",
+    "Dark Purple": "M\xf8rkelilla",
+    "Dark Blue": "M\xf8rkebl\xe5",
     "Black": "Svart",
     "White": "Hvit",
     "Switch to or from fullscreen mode": "Bytt til eller fra fullskjermmodus",
-- 
2.39.2 (Apple Git-143)

